WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -66,10 +69,9 @@ class PageListViewModel @Inject constructor(
     val pages: LiveData<Triple<List<PageItem>, Boolean, Boolean>> = Transformations.map(_pages) {
         Triple(it, isSitePhotonCapable, isSitePrivateAt)
     }
-
+    var quickStartEvent: QuickStartEvent? = null
```

Comment:
We could probably remove this from here and keep just the `QuickStartTask` in our state